requirements.txt

```
@@ -1,5 +1,5 @@
 numpy
-setuptools >= 41
-opencv-python-headless
-ocrd >= 2.22.3
-tensorflow >= 2.4.0
+setuptools>=41
+ocrd>=2.22.3
+opencv-python-headless==4.6.0.66
+tensorflow==2.6.2
```

Same issue from ocrd_all's perspective. We do set minimum versions for tensorflow and in fact, tensorflow 2.6.2 is currently used (except for the tensorflow processors still on 1.15.x) but for the same reason as for opencv, we want to support the widest possible range of supported versions.
```suggestion
tensorflow>=2.4
```

---

I think supporting later versions of tensorflow is a good idea. We should just ensure that the binarization model that is provided for downloading is actually compatible to those versions (spoiler: the models downloaded from https://qurator-data.de/sbb_binarization/ are not - I had to open and re-save them in TF 2.6.2 for being able to use them in TF 2.9). That's why I proposed to pin TF to a version where the current model definitely works.

---

But it should work with 2.4.0 upwards? eynollah does specify `>= 2.4.0` as well, calamari depends on `>= 2.5.0`, actually installed currently is `2.6.2`. Unless 2.4 does not work with the current models anymore, I would prefer to keep the minimum version. If we do need to upgrade the dependency, I would want to coordinate the change across projects.

> spoiler: the models downloaded from https://qurator-data.de/sbb_binarization/ are not - I had to open and re-save them in TF 2.6.2 for being able to use them in TF 2.9

@vahidrezanezhad @mikegerber Maybe it would make sense to provide the loaded-updated-and-saved models for later tensorflow versions via qurator-data.de? We have a (currently unused) `version_range` mechanism in the resource manager for this purpose and this would be a good use case to test it AFAICT.

---

I've made the converted models in the (more) portable TF format available here, if you are interested: https://github.com/apacha/sbb_binarization/releases/tag/pre-trained-models